platform/ui/src/components/userPreferencesModal/AboutModal.js

```diff
@@ -1,31 +1,17 @@
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import Modal from 'react-bootstrap-modal';
```

Comment:
This should no longer need this as a dependency.

Reply:
I'm still using react-boostrap-modal in this branch, I decided to rollback for now because it has its builtin style schema already defined like body, header, footer.